release.yaml

```diff
@@ -29,7 +29,7 @@ managers:
     isolated: true
     
   - artifact: dev.galasa.cicsts.manager
-    version: 0.25.0
+    version: 0.26.0
```

Comment:
why do we need to bump the cicsts manager?

Reply:
We have updated CicstsManagerImpl so need to bump the version of the Manager